docs/api/spec/components/schemas/ipset.yaml

```diff
@@ -0,0 +1,48 @@
+IpsetNetwork:
+    type: object
+    properties:
+        network:
+            type: string
+            description: The IPv4 address of the network
```

Comment:
I think it should be 'The IPv4 network identifier' since I'm not sure its technically an address